Create issue templates

stevepiercy opened the pull request:
This PR adds issue templates. They're basic and can be overridden per repo.

A good demo can be found at https://github.com/plone/volto/issues/new/choose.

See https://github.com/plone/.github/community to see current community health status.

stevepiercy committed:
Create issue templates

Create config.yml

stevepiercy committed:
Create 1_bug_report.md

stevepiercy committed:
Create 2_feature_request.md

stevepiercy committed:
Create 3_PLIP.md

dataflake commented:
We have a pretty simple template at https://github.com/zopefoundation/.github/blob/master/.github/ISSUE_TEMPLATE/Bug_report.md?plain=1, just for comparing.

One item we have on ours that is missing here: Ask people to email security-related reports to security@plone.org.

I noticed that your configuration explicitly prevents choosing a blank issue. That's a valid choice for users who are not very familiar with the development process. Me personally I would be annoyed. I find myself always choosing the blank issue template for our own projects because it causes the least friction for me as a maintainer.

stevepiercy committed:
Update config.yml

- Allow blank issues, adding comment about the this option.
- Add link to report security issues.

stevepiercy commented:
@dataflake thank you again for the feedback. I made a few changes accordingly in https://github.com/plone/.github/pull/11/commits/79808b6d7fcff88685e96af533a4fb1f9690edb0.

dataflake commented:
I would also add an explicit instruction to file security-related reports by email to security@plone.org into the bug report template text.

stevepiercy committed:
Add admonition about reporting security issues

stevepiercy commented:
> I would also add an explicit instruction to file security-related reports by email to [security@plone.org](mailto:security@plone.org) into the bug report template text.

@dataflake I added an admonition in https://github.com/plone/.github/pull/11/commits/61990182c28914fe7e5bc832c43a25e2873d7918. One more review, please? Thank you!

dataflake reviewed (approved):
Thank you for doing this work. Housekeeping is a thankless job most of the time.
